Drop central definition of $portmap, it's better served by the individual packages providing $portmap.
--- a/insserv.conf
+++ b/insserv.conf
@@ -26,11 +26,6 @@ $remote_fs	$local_fs +mountnfs +mountnfs
 $syslog		+rsyslog +sysklogd +syslog-ng +dsyslog +inetutils-syslogd
 
 #
-# SunRPC portmapper available
-#
-$portmap	portmap
-
-#
 # The system time has been set correctly
 #
 $time		+hwclock
